test/e2e/zookeepercluster_test.go

```diff
@@ -57,9 +57,9 @@ func testZookeeperCluster(t *testing.T) {
 	}
 
 	testFuncs := map[string]func(t *testing.T){
+		"testUpgradeCluster":        testUpgradeCluster,
```

Comment:
Is there a reason for moving this on top?

Reply:
There's no specific reason, I have changed it back.

Reply:
cool